src/main.cpp

```diff
@@ -0,0 +1,52 @@
+#include "frontends/common/parseInput.h"
```

Comment:
I removed all the non-essential code from the main file. It only needs to access the API of the other compilation units.